OpenSim/Simulation/Model/CoordinateLimitForce.h

```diff
@@ -235,9 +237,10 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoordinateLimitForce, Force);
 
     // Smooth step functions for continuous transition from no stiffness and 
     // damping to constant values beyond the limits. These are heap allocated
-    // and owned here so must be deleted in destructor.
-    SimTK::ReferencePtr<SimTK::Function::Step> _upStep;
-    SimTK::ReferencePtr<SimTK::Function::Step> _loStep;
+    // and owned here, but we don't want these functions to be copied if a
+    // CoordinateLimitForce is copied.
```

Comment:
> Oh, is it because the weighting comes from the model?

Yes; it looks like the function parameters depend on whether the coordinate is rotational or translational.